onyo/lib/commands.py

```diff
@@ -1048,7 +1048,8 @@ def onyo_rm(inventory: Inventory,
         if p.name in [OnyoRepo.ANCHOR_FILE_NAME, OnyoRepo.ASSET_DIR_FILE_NAME]:
             raise InvalidArgumentError(f"Cannot remove onyo-managed files ({p}).\n"
                                        f"You may want to remove {p.parent} instead.")
-        if not item['onyo.is.asset'] and not item['onyo.is.directory']:
+        if (not item['onyo.is.asset'] and not item['onyo.is.directory']) or \
+                item['onyo.is.template']:
```

Comment:
Ahh, are we keeping `'onyo.is.template'`? (For now at least)?

Reply:
Yes. Not sure whether it's going to be useful really. But didn't want to prematurely remove it.